images/base/files/usr/local/bin/entrypoint

```diff
@@ -19,7 +19,7 @@ set -o nounset
 set -o pipefail
 
 fix_mount() {
-  echo 'INFO: ensuring we can execute /bin/mount even with userns-remap'
+  echo 'INFO: ensuring we can execute /bin/mount even with userns-remap' 
```

Comment:
Trailing whitespace?

Reply:
🤷‍♂️ this won't be the last chance to this file and we bumped the image